src/vendor/jsPlumb.js

```diff
@@ -8820,34 +8820,32 @@ var JSPLUMB_ZOOM = 1;
                 return a;
             },
         // used by edgeSortFunctions
-            currySort = function (reverseAngles) {
+            currySort = function (flipValues) {
                 return function (a, b) {
-                    var r = true;
-                    if (reverseAngles) {
-                        r = a[0][0] < b[0][0];
-                    }
-                    else {
-                        r = a[0][0] > b[0][0];
+                    var p1 = a[0][0],
+                        p2 = b[0][0],
+                        result;
+                    if (flipValues) {
+                        // normalize the values on the negative y axis, which flips from -Pi to Pi when rotating counterclockwise
+                        p1 = p1 < 0 ? -Math.PI - p1 : Math.PI - p1,
+                        p2 = p2 < 0 ? -Math.PI - p2 : Math.PI - p2;
+                        // reversed on the left ride compared to the right side
```

Comment:
```suggestion
                        // reversed on the left side compared to the right side
```

Reply:
Hee hee.  I put this in to see if you would catch it @kswenson :)